styles.css

```
@@ -0,0 +1,290 @@
+   list-style-type: none;
+   -webkit-font-smoothing: antialiased;
+   transform-origin: 0% 0%;
+   transform: translate(-100%, 0);
+   font-family: 'GTWalsheim','';
+   transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0);
+   list-style-type: none;
+ }
+ 
+.menu-items li{
+   padding: 10px 0;
+   font-size: 1.2em;
+   text-align: center;
+   font-weight: bold;
+ }
+
+
+
+.menu-items li a,  .phone-link{
+   text-decoration: none;
+   color: #00343D;
+   
+ }
+.menu-items li a:hover,.menu-items li a:active,
+ .phone-link:hover, .phone-link:active{
+   font-style: italic;
+ }
+
+#nav-items .contactLink{
+  margin-top: 50px;
+}
+
+#nav-items .contactLink a{
+  background: #0BD8A2;
+  border-radius: 50px;
+  padding: 10px 25px;
+}
+
+.menu-items.open{
+   transform: none;
+ }
+
+ /*-------------fin seccion de NAV --------------------*/
+
+ .main-header{
+   text-align: center;
+ }
+
+ h1 {
+  margin-top: 60px;
+  font-size: 1.6em;
+ }
+
+ .main-header button{
+  background: #fff;
+  border-radius: 50px;
+  padding: 10px 25px;
+  border: 0;
+  font-weight: bold;
+  font-size: 1.1em;
+  margin: 40px 0;
+ }
+
+ .main-header img{
+  width: 80%;
+  margin: 0 10%;
+  height: 300px;
+  object-fit: contain;
+ }
+
+ @media only screen and (min-width: 500px) {
+  body{
+    background-image: url("./images/header.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  /*------------------------------------nav------------------------------------------*/
+  .menu-items-toggle, .home-logo, .phone-link img{
+    filter: grayscale(0) brightness(0);
+    /*fill: #00D8A0;*/
+  }
+
+  .home-logo .logoBig{
+    display: block;
+  }
+  .home-logo .logoSmall{
+    display: none;
+  }
+
+  /*------------------------------------header------------------------------------------*/
+ .main-header{
+    text-align: left;
+    padding: 0 5%;
+  }
+
+  h1 {
+    margin-top: 40px;
+    font-size: 2.3em;
+    width: 50%;
+  }
+
+  .main-header img{
+    position: absolute;
+    z-index: -1;
+    right: 0;
+    top: 100px;
+    width: auto;
+    margin: 0 5%;
+  }
+    
+}
+
+
+@media only screen and (min-width: 834px) {
+  body{
+    background-image: url("./images/header-big.png");
+  }
+
+  /*------------------------------------nav------------------------------------------*/
+  .burger-icon{
+    display: none;
+  }
+  .menu-items{
+    position: relative;
+    display: inline-flex;
+    justify-content: flex-end;
+    margin: 0;
+    padding: 0;
+    background: none;
+    transform: none;
+  }
+  .menu-items li{
+    display:inline;
+    margin: 0 2%;
+  }
+  .menu-items-toggle {
+    display: none;
+  }
+  #nav-items .contactLink a{
+    background: none;
+    padding: 0;
+  }
+  #nav-items .contactLink{
+    margin-top: 0;
+    margin-left: 40px;
+  }
+
+  /*------------------------------------header------------------------------------------*/
+  .main-header{
+    padding: 0 10%;
+  }
+  .main-header img{display: none;}
```

Avoid using one-line rules. In that way your code readability will improve
```suggestion
  .main-header img {
    display: none;
  }
```